### PR TITLE
fix(database-change): sql plan task execute file script failed

### PR DESCRIPTION
**Pull request by kiko-art**
#### What type of PR is this?
type-bug

#### What this PR does / why we need it:
1 .Fixed a bug where sql attachment execution failed.
2.Sql attachment display added to the Task Details page.
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java [hunk 1]
```diff
@@ -194,7 +195,14 @@ private void initSqlIterator() {
         }
 
         for (String sqlObjectId : parameters.getSqlObjectIds()) {
-            try (InputStream current = cloudObjectStorageService.getObject(sqlObjectId)) {
+            InputStream inputStream;
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java:198]: when to close this stream, may add comments

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java [hunk 2]
```diff
@@ -207,7 +215,7 @@ private void initSqlIterator() {
                 }
                 sqlInputStream = new SequenceInputStream(sqlInputStream, current);
             } catch (IOException e) {
-                log.warn("Read content from cloud object storage failed, sqlObjectId={}", sqlObjectId);
+                log.warn("Parsing file failed, objectName={}", sqlObjectId);
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java:218]: add exception info here
` log.warn("Parsing file failed, objectName={}, errorReason={}", sqlObjectId, ExceptionUtils.simpleReason(e));`

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java [hunk 3]
```diff
@@ -108,7 +109,7 @@ protected void doInit(JobContext context) {
                 PublishSqlPlanJobReq.class);
         JobContext jobContext = getJobContext();
         Map<String, String> jobProperties = jobContext.getJobProperties();
-        this.result.setRegion(jobProperties.get("region"));
+        this.result.setRegion(jobProperties.get("regionName"));
```
Comment (guowl3) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java:112]: jobProperties.get("regionName") -> JobPropertiesUtils.getRegionName(jobProperties)

Comment (kiko-art) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java]: Have been changed.

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java (hunk 2 above)
Comment (kiko-art) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java]: Have been resolved.

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java (hunk 1 above)
Comment (kiko-art) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java]: Have been resolved.

**Review by yizhouxw — APPROVED** [review]
lgtm

**Review by guowl3 — APPROVED** [review]
lgtm

**Pull request merged**